### PR TITLE
Add new article Music and silence in coding

### PR DESCRIPTION
**Pull request by SHABA-Rex**
Hello everyone, thank you in advance for your verification.

### CODE REVIEW

File: content/post/articles/Music-and-silence.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Creativity can be boosted by music, according to a [study](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0182210) by researchers at Radboud University (Netherlands) and Sydney University of Technology (Australia) the researchers found that listening to happy music, which they define as music with positive, awakening values (a relative positivity because coders who love devil metal are also happy) facilitates more inspiring creative thinking than silence.
+
+I'm a supporter of this opinion, so here are some nuggets(without wanting to advertise) I use when I meet mister bug:
+- [RISE (ft. The Glitch Mob, Mako, and The Word Alive)](https://www.youtube.com/watch?v=fB8TyLTD7EE)
+- [Phoenix (avec Cailin Russo et Chrissy Costanza)](https://www.youtube.com/watch?v=i1IKnWDecwA)
+- [Take Over (avec Jeremy McKinnon (A Day To Remember), MAX, Henry)](https://www.youtube.com/watch?v=KbNL9ZyB49c)
+
+# Code with silence
+
+Lao Tzu said <<*Silence is a source of Great Strength*>>
```
Comment (RMPR) [inline content/post/articles/Music-and-silence.md:36]: I don't think this quote fits the whole point of this section. Not only [it's not directly from Lao Tzu](https://www.taoistic.com/fake-laotzu-quotes/fake-laotzu-quote-Silence_is_a_source_of_great_strength.htm) but where it's coming from, (The Tao of Leadership: Lao Tzu's Tao Te Ching Adapted for a New Age from 1985, by John Heider), where the author allowed himself considerable deviations from the original. If we quote including the 4 lines before the excerpt you took:

> The leader does not gossip about others or waste breath arguing the merits of competing theories.
Silence is a great source of strength.

Here silence is not meant for silence around you, rather silence from you (or in you as you wish), so I don't think it's really applicable in this specific case.

File: content/post/articles/Music-and-silence.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Not everyone uses music to cope with the different difficulties that one can encounter while coding so there are those who are for silence (the zone), a 2013 [study](https://www.researchgate.net/publication/259110014_Is_silence_golden_Effects_of_auditory_stimuli_and_their_absence_on_adult_hippocampal_neurogenesis) found that two hours of silence could create new cells in the hippocampus region, a brain area linked to learning, remembering, and emotions.According to a 2006 [study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1860846/), two minutes of silence relieves tension in the body and brain and is more relaxing than listening to music.
```
Comment (RMPR) [inline content/post/articles/Music-and-silence.md:38]:
```suggestion
Not everyone uses music to cope with the different difficulties that one can encounter while coding so there are those who are for silence (the zone), a 2013 [study](https://www.researchgate.net/publication/259110014_Is_silence_golden_Effects_of_auditory_stimuli_and_their_absence_on_adult_hippocampal_neurogenesis) found that two hours of silence could create new cells in the hippocampus region, a brain area linked to learning, remembering and emotions. According to a 2006 [study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1860846/), two minutes of silence relieves tension in the body and brain and is more relaxing than listening to music.  
```

You should mention that this [study](https://www.researchgate.net/publication/259110014_Is_silence_golden_Effects_of_auditory_stimuli_and_their_absence_on_adult_hippocampal_neurogenesis) was conducted on mice, and the type of music involved is again, classical music. 

And this:
> [study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1860846/), two minutes of silence relieves tension in the body and brain and is more relaxing than listening to music.  

Doesn't paint the complete picture, the articles specifically states that two minutes of silence *between* two music listening session is what causes relaxation. There are no mention of silence alone. So this is more an argument for both music and silence, with no clear cut.

File: content/post/articles/Music-and-silence.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Not everyone uses music to cope with the different difficulties that one can encounter while coding so there are those who are for silence (the zone), a 2013 [study](https://www.researchgate.net/publication/259110014_Is_silence_golden_Effects_of_auditory_stimuli_and_their_absence_on_adult_hippocampal_neurogenesis) found that two hours of silence could create new cells in the hippocampus region, a brain area linked to learning, remembering, and emotions.According to a 2006 [study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1860846/), two minutes of silence relieves tension in the body and brain and is more relaxing than listening to music.  
+Silence also has its advantages:
+
+###### Boost creativity
+When you allow thoughts to go where they want, inspiration can flow. Solutions to current or long standing problems may suddenly appear to you, or a workaround or innovative approach may seem more achievable. Ideas for going in a different direction might merge together, help build momentum and enthusiasm to turn them into other potential avenues to pursue.Follow this [link](https://psychcentral.com/blog/the-hidden-benefits-of-silence#1) to read the article.
```
Comment (RMPR) [inline content/post/articles/Music-and-silence.md:42]: A so-called scientific website with ads all over the place makes me question his legitimacy, moreover it starts with the quote:

> “Silence is a source of great strength.” – Lao Tzu

That we already established is not from Lao Tzu. It makes me question how deep they went in their research. Nonetheless, to their credit, they say:

> Silence may help in the following areas

"may" is a keyword here, that means it's possible, not for sure, but possible. Moreover, nowhere in the article they talk about silence during coding. There, silence is more associated to a break, which is another subject by its own.

File: content/post/articles/Music-and-silence.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+Silence also has its advantages:
+
+###### Boost creativity
+When you allow thoughts to go where they want, inspiration can flow. Solutions to current or long standing problems may suddenly appear to you, or a workaround or innovative approach may seem more achievable. Ideas for going in a different direction might merge together, help build momentum and enthusiasm to turn them into other potential avenues to pursue.Follow this [link](https://psychcentral.com/blog/the-hidden-benefits-of-silence#1) to read the article.  
+
+###### Deal with your emotions
+When you practice solitude and silence, you allow your brain to process your emotions, rather than suppressing them and staying on autopilot. You may experience uncomfortable emotions during this process, but the sooner you recognize and process your feelings, the sooner you can move forward in health and freedom.
```
Comment (RMPR) [inline content/post/articles/Music-and-silence.md:45]: Now I am a bit confused, the title is "Music and silence *in coding* " why are you talking of the "practice of solitude and silence"? What you're saying here is not directly related (if at all) to coding, or at least, I am not able to see it.

File: content/post/articles/Music-and-silence.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+When you allow thoughts to go where they want, inspiration can flow. Solutions to current or long standing problems may suddenly appear to you, or a workaround or innovative approach may seem more achievable. Ideas for going in a different direction might merge together, help build momentum and enthusiasm to turn them into other potential avenues to pursue.Follow this [link](https://psychcentral.com/blog/the-hidden-benefits-of-silence#1) to read the article.  
+
+###### Deal with your emotions
+When you practice solitude and silence, you allow your brain to process your emotions, rather than suppressing them and staying on autopilot. You may experience uncomfortable emotions during this process, but the sooner you recognize and process your feelings, the sooner you can move forward in health and freedom.  
+Follow this [link](https://www.piedmont.org/living-better/the-benefits-of-silence-and-solitude) to read the article.
+
+
+In the end, I think that to improve creativity, concentration when coding, it really depends on everyone's desires and affinities, whether with music or with silence (the zone)🔥.  
+
+Thanks for reading. Here are my sources:
```
Comment (RMPR) [inline content/post/articles/Music-and-silence.md:51]: Mentioning articles you already linked in the body of the article is a bit redundant, don't you think?

**Review by RMPR — CHANGES_REQUESTED** [review]

File: content/post/articles/Music-and-silence.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+date: 2021-12-13T15:00:48+01:00
+images: ["/shaba-rex.png"]
+categories: ["Software development"]
+tags: ["Coding","Music","Silence","Software development","CaParleDev","Bugs"]
+authors: ["Shaba-Rex"]
+---
+{{<figure src="/Music-and-silence-in-coding/Listen-to-music.png"  alt="Liste to music while coding">}}
+
+Listen to music while coding or enter the zone (state of concentration created by silence).
+Opinions are divided on the subject, on the one hand we have coders who like to boost their concentration and productivity by listening to music and those who only want "silence"👄, in this article we will explore these two opinions.
```
[the remark this answers is not in the document]
Comment (SHABA-Rex) [reply content/post/articles/Music-and-silence.md:12]: Hello, thank you for your review.
I put the emojis to give more emotions to my words, to make them a little more visual , and i think is important to visualise while reading the article.
My editor didn't have this emoji, thanks to you, I'll be able to copy the one you put in your comment. :) 
Please do you know how to put this emoji 🤫 on sublime text without copying it on something?

File: content/post/articles/Music-and-silence.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+# Code with music
+Friedrich Nietzsche said <<*Without music life would be a mistake.*>>
+
+The brain has to make efforts to analyze and process information based on abstract and logical reasoning during coding, several studies such as [University of Birmingham research](http://www.sciencedirect.com/science/article/pii/0003687072901019), [PNAS research](https://www.pnas.org/content/116/8/3310), [Stanford University School of Medicine research](https://www.sciencedirect.com/science/article/pii/S0896627307005004) show that music is effective in improving focus and productivity.
```
[the remark this answers is not in the document]
Comment (SHABA-Rex) [reply content/post/articles/Music-and-silence.md:17]: In this paragraph, my objective was to state that music has a good impact on our brain activity when coding regardless of the type of music or the way of coding and to show that this impact would improve concentration and productivity. I agree with you that the studies chosen were not appropriate to support this view.
I will replace those studies by a good one.

**Pull request closed without merging**